crates/transaction-emitter-lib/src/transaction_generator/nft_mint_and_transfer.rs

```diff
@@ -31,27 +38,52 @@ impl NFTMint {
             creator_account,
             collection_name,
             token_name,
+            nft_balance: HashMap::new(),
         }
     }
 }
 
-impl TransactionGenerator for NFTMint {
+impl TransactionGenerator for NFTMintAndTransfer {
     fn generate_transactions(
         &mut self,
         accounts: Vec<&mut LocalAccount>,
         transactions_per_account: usize,
     ) -> Vec<SignedTransaction> {
         let mut requests = Vec::with_capacity(accounts.len() * transactions_per_account);
+        let mut visited_accounts: Vec<&LocalAccount> = vec![];
         for account in accounts {
```

Comment:
https://github.com/aptos-labs/aptos-core/blob/9deb340d1e35ab5b2f46b9e984bceb1968d46243/crates/transaction-emitter-lib/src/transaction_generator/p2p_transaction_generator.rs#L153-L155

^ to gain concurrency, the p2p emitter, for example, transfers from each of  `accounts` to a random receiver out of `all_receivers`

Reply:
We need a `LocalAccount` to sign...